secator/runners/command.py

```
@@ -3,6 +3,7 @@
 import logging
 import os
 import re
+import resource
```

Ruff (0.12.2)

`resource` imported but unused

Remove unused import: `resource`

(F401)